wdl/GenotypeDepthPart2.wdl

```diff
@@ -221,9 +223,10 @@ task IntegrateDepthGq {
 task GetRegenotype{
   input{
     File depth_genotyped_vcf
-    File? cohort_combined_bed
-    File? cohort_sort
+    File? regeno_sample_counts_lookup
+    File? regeno_raw_combined_depth
     String Batch
+    Int? n_samples_cohort
```

Comment:
Even though `n_samples_cohort` will always be defined as the workflow is written, it is good practice to make it non-optional, since the command section would not run correctly without it. In the call, you will need to force it using `n_samples_cohort=select_first([n_samples_cohort])`

Reply:
Also, I realize you found it this way, but the same applies to `regeno_sample_counts_lookup` and `regeno_raw_combined_depth`

Reply:
Good point. Done